src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String DATA_SEPARATOR = " ";
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int NUMBER_OF_DAYS_FOR_BORDER_SHIFT = 1;
+    private Report report;
```

Comment:
Here I would recommend to make `report` variable local. In such way, we can achieve more predictability in our code and better encapsulation.

Reply:
Fixed